training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Same suggestion for the intro paragraph:</p><p> </p><p>You'll learn how to train the GPT-2 model with SageMaker's model parallelism.</p><p>--&gt;</p><p>You'll learn how to run FP16 training of the GPT-2 model with SageMaker's model parallelism.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>